kola/tests/rpmostree/deployments.go

```diff
@@ -239,7 +239,7 @@ func rpmOstreeUpgradeRollback(c cluster.TestCluster) {
 // currently using.  We've already had to swap from `fpaste` to `bcrypt`
 func rpmOstreeInstallUninstall(c cluster.TestCluster) {
 	var installPkgName = "bcrypt"
-	var installPkgBin = "/bin/bcrypt"
+	installPkgBin := [2]string{"/bin/bcrypt", "/usr/bin/bcrypt"}
```

Comment:
Lets instead set this based on which distro we're on. It would also keep the logic below simpler.